src/daft-schema/src/dtype.rs

```diff
@@ -102,16 +102,16 @@ pub enum DataType {
 #[derive(Serialize, Deserialize)]
 struct DataTypePayload {
     datatype: DataType,
-    daft_version: String,
-    daft_build_type: String,
+    // daft_version: String,
+    // daft_build_type: String,
 }
 
 impl DataTypePayload {
     pub fn new(datatype: &DataType) -> Self {
         DataTypePayload {
             datatype: datatype.clone(),
-            daft_version: crate::VERSION.into(),
-            daft_build_type: crate::DAFT_BUILD_TYPE.into(),
+            // daft_version: crate::VERSION.into(),
+            // daft_build_type: crate::DAFT_BUILD_TYPE.into(),
```

Comment:
What's this and why is it commented out?

Reply:
Ah these were declared in daft-core (Daft-version and build type). factoring these out to a common crate.